main.js

```diff
@@ -0,0 +1,252 @@
+};
+class Player {
+  constructor(symbol) {
+    this.symbol = symbol;
+    // NOT WORKING
+    // this.array = [];
+  }
+
+  play1() {
+    if (checkPositions().position1 != "X" && checkPositions().position1 != "O") {
+      game.board[0][0] = this.symbol;
+      counter++;
+      // this.array.push(1);
+      // if (checkWin(combo1, this.array)) {
+      //   console.log("Winner");
+      // }
+      game.checkWinner(game.board);
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play2() {
+    if (checkPositions().position2 != "X" && checkPositions().position2 != "O") {
+      game.board[0][1] = this.symbol;
+      counter++;
+      // this.array.push(2);
+      game.checkWinner(game.board);
+
+    gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play3() {
+    if (checkPositions().position3 != "X" && checkPositions().position3 != "O") {
+      game.board[0][2] = this.symbol;
+      counter++;
+      // this.array.push(3);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play4() {
+    if (checkPositions().position4 != "X" && checkPositions().position4 != "O") {
+      game.board[1][0] = this.symbol;
+      counter++;
+      // this.array.push(4);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play5() {
+    if (checkPositions().position5 != "X" && checkPositions().position5 != "O") {
+      game.board[1][1] = this.symbol;
+      counter++;
+      // this.array.push(5);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play6() {
+    if (checkPositions().position6 != "X" && checkPositions().position6 != "O") {
+      game.board[1][2] = this.symbol;
+      counter++;
+      // this.array.push(6);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play7() {
+    if (checkPositions().position7 != "X" && checkPositions().position7 != "O") {
+      game.board[2][0] = this.symbol;
+      counter++;
+      // this.array.push(7);
+      game.checkWinner(game.board);
+
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play8() {
+    if (checkPositions().position8 != "X" && checkPositions().position8 != "O") {
+      game.board[2][1] = this.symbol;
+      counter++;
+      // this.array.push(8);
+      game.checkWinner(game.board);
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+  play9() {
+    if (checkPositions().position9 != "X" && checkPositions().position9 != "O") {
+      game.board[2][2] = this.symbol;
+      counter++;
+      // this.array.push(9);
+      game.checkWinner(game.board);
+      gameState();
+    } else {
+      moveTaken();
+    }
+  }
+
+};
+
+var gameState = function() {
+    console.log("This is the current game board: " + "\n" + game.board[0] + "\n" + game.board[1] + "\n" + game.board[2]);
+};
+
+var moveTaken = function(){
```

Comment:
I think the only reason why you need this to be it's own function is because of how many times you're repeating the "play" logic. If you had it just once, you wouldn't need all of this.